src/dist.config.ts

```diff
@@ -49,7 +49,7 @@ function webpackConfig(args: any): webpack.Configuration {
 		}),
 		new HtmlWebpackPlugin({
 			inject: true,
-			chunks: args.singleBundle ? ['main'] : ['runtime', 'main'],
+			chunks: args.singleBundle ? ['main'] : ['runtime', 'bootstrap'],
```

Comment:
we no longer need a `runtime` i don't think, `bootstrap` will change so infrequently I think we can merge them

Reply:
No problemo